wordpress-mobile/WordPress-Android
# Feature: Feedback form attachments

This is the feature branch for uploading feedback form attachments. It contains the following child PRs:

* #21107
* #21115
* #21116 
* #21117 

To test:

1. Tap Help on the Me page and ensure the "Contact email" is correct (this for step 6)
2. Return to the Me page and tap "Send feedback"
3. Enter a message
4. Add a couple of attachments
5. Tap submit and verify it all works
6. When you get the email about the ticket, go to https://a8c.zendesk.com/agent/tickets/$ticketNumber and verify the attachments were added

Caveats:

- Unlike iOS, there is no preview of attachments yet - that's being worked on separately in [this branch](https://github.com/wordpress-mobile/WordPress-Android/pull/21121)
- Like iOS, there isn't a way to cancel the attachment uploading. I suspect we'll want to add this at some point.
